src/app/shared/models/config/config.interface.ts

```diff
@@ -22,6 +22,13 @@ export interface CustomizableConfig {
   sessionRefreshInterval: number;
   apiDocLink: string;
   extensions: ExtensionsConfig;
+  /*
+  *  Log View settings
+  * */
+  vmLogs: {
+    autoUpdateRefreshFrequency: number;
+    autoUpdateRequestedInterval: number;
```

Comment:
If you add new customizable properties, it will be great to add a description for this configuration in the config-guide